manual/src/tutorials/parallelism.etex

```diff
@@ -0,0 +1,690 @@
+\chapter{Parallel programming}
+%HEVEA\cutname{parallelism.html}
+\label{c:parallelism}
+
+In this chapter, we shall look at the parallel programming facilities in OCaml.
+The OCaml standard library exposes low-level primitives for parallel
+programming. We recommend the users to utilise higher-level parallel
+programming libraries such as
+\href{https://github.com/ocaml-multicore/domainslib}{domainslib}. This
+tutorial will first cover the high-level parallel programming using domainslib
+followed by low-level primitives exposed by the compiler.
+
+OCaml distinguishes concurrency and parallelism and provides distinct
+mechanisms for expressing them. Concurrency is overlapped execution of tasks
```

Comment:
Can you `\ref` into the main manual here? Section `s:effects-concurrency` has a nice definition of exactly what a task is and the states it can be in.

Reply:
Done.